homepairs/homepairsUI/src/state/account/actions.tsx

```diff
@@ -148,10 +148,10 @@ export const fetchAccount = (
             password: Password,
           })
           .then((response) => {
+            console.log(response);
```

Comment:
we all use this console.log a lot i believe, so let's keep it, but commenting now so we eventually remember to get rid of it

Reply:
(same for line 172 and 181 / other logged responses)